v20.1/cockroach-nodelocal-upload.md

```diff
@@ -0,0 +1,90 @@
+---
+title: cockroach nodelocal upload
+summary: Upload a file to a node's local file system.
+toc: true
+---
+
+<span class="version-tag">New in v20.1:</span> The `cockroach nodelocal upload` [command](cockroach-commands.html) uploads a file to a node's (the gateway node, by default) local file system.
```

Comment:
ditto above -- i'm include right away that it is uploading _to the external io dir_ in the introduction. I know it is a detail and that it is covered in the next paragraph, but arbitrary file access is scary and without that "node's local filesystem" makes it sounds like it is wide open. Someone might panic here instead of reading the details.